app/scripts/views/marketing_snippet.js

```diff
@@ -130,6 +143,20 @@ define(function (require, exports, module) {
       return this._getUap().isAndroid();
     },
 
+    /**
+     * Add the appropriate metrics related query params to
+     * the Firefox app's app store link.
+     *
+     * @param {String} base
+     * @returns {String}
+     */
+    _storeLink (base) {
+      return Strings.interpolate(base, {
+        campaign: 'fxa-conf-page',
+        creative: View.BUTTON_IDS[ this._marketingId ],
```

Comment:
nit: the spaces here and below are a bit strange ( `[ this._marketingId ]`) . A bit of inconsistency with the rest of the code base. example, we have `var command = this.commands[commandName];` in fx-sync.